code/framework/api/src/main/java/io/cattle/platform/api/auth/Identity.java

```diff
@@ -0,0 +1,97 @@
+package io.cattle.platform.api.auth;
+
+import io.github.ibuildthecloud.gdapi.annotation.Field;
+import io.github.ibuildthecloud.gdapi.annotation.Type;
+
+@Type(name = "identity", pluralName = "identities")
+public class Identity {
+
+    private final String externalId;
+    private final String profilePicture;
+    private final String name;
+    private final String kind;
+    private final String profileUrl;
```

Comment:
They should be both then shouldn't they? Because they are properties on the resource not links to more of cattle. They are for the ui to be able to display information about this identity.